### ISSUE TITLE
Re-add support for postgis-sfcgal / protobuf-backed functions

### ISSUE THREAD

**Issue by JamesChevalier** [PostgresApp/PostgresApp#785]
[v2.7.7](https://github.com/PostgresApp/PostgresApp/releases/tag/v2.7.7) included this known issue:

> PostGIS does not currently ship with support for `postgis-sfcgal` and `protobuf`-backed functions like `ST_AsMVT()` and `ST_AsGeobuf()`.

[v2.7.8](https://github.com/PostgresApp/PostgresApp/releases/tag/v2.7.8) was released without a resolution to that issue.

The only issue that mentions ST_AsMVT or protobuf is an old one from 2017, so I wanted to open this for that task.

As best I can tell, https://github.com/PostgresApp/PostgresApp/commit/786f30c7f3b93bb1a6385cebbaddf8a9159cd416 is the only commit that references this temporary removal, noting:

> linking is currently not working, further investigation needed

**Comment by JamesChevalier** [comment]
In the meantime, to work around this issue and also the issue that the official PostGIS Docker image only supports x86-64 architecture 😅 I created my own `Dockerfile`:

```
FROM postgres:17

ENV POSTGRES_HOST_AUTH_METHOD=trust

RUN apt-get update \
  && apt-get install -y --no-install-recommends postgresql-17-postgis-3 postgresql-17-postgis-3-scripts \
  && rm -rf /var/lib/apt/lists/*
```

Building it is pretty standard: `docker build -t postgis .`
Running it is similarly standard: `docker run --name postgres17 -d -v postgres17:/var/lib/postgresql/data -p 5432:5432 postgis:latest`

**Comment by tbussmann** [comment]
Thank you for your interest in these features. It is true that the current releases are missing these features of PostGIS with PostgreSQL 17. In the meantime, `postgis-sfcgal` has been worked on (see 03b4817) and will be included in next week's releases again. I'm also working on issues with the `protobuf-c` dependencies (`protobuf` with `abseil` and `bazel`) which will fix the missing MVT in PostGIS 3.5.

**Comment by tbussmann** [comment]
This should be resolved with Postgres.App [v2.7.9](https://github.com/PostgresApp/PostgresApp/releases/tag/v2.7.9) released today.
